packages/earn-controller/src/index.ts

```diff
@@ -1,7 +1,9 @@
 export type {
   PooledStakingState,
-  StablecoinLendingState,
-  StablecoinVault,
+  LendingState,
```

Comment:
Should we document these changes in the changelog? Specifically it seems like these were removed (i.e. these are breaking changes):

- `StablecoinLendingState`
- `StablecoinVault`

and these were added:

- `LendingState`
- `LendingMarketWithPosition`
- `LendingPositionWithMarket`
- `LendingPositionWithMarketReference`
- `selectLendingMarkets`
- `selectLendingPositions`
- `selectLendingMarketsWithPosition`
- `selectLendingPositionsByProtocol`
- `selectLendingMarketByProtocolAndTokenAddress`
- `selectLendingMarketForProtocolAndTokenAddress`
- `selectLendingPositionsByChainId`
- `selectLendingMarketsByChainId`
- `selectLendingMarketsByProtocolAndId`
- `selectLendingMarketForProtocolAndId`
- `selectLendingPositionsWithMarket`
- `selectLendingMarketsForChainId`